codegen/arrai/grpc_client.arrai

```diff
@@ -0,0 +1,98 @@
+# https://github.com/anz-bank/sysl/blob/master/pkg/sysl/sysl.proto
+# Sysl data structure to handle
+\sysl (
+    grpcclient: \a \name \proj_location
+        # sysl util (move out to helper function in the future)
+        let jsontool = (
```

Comment:
Move json tool to new file.

Reply:
Im left it in here for now because the arrai imports didnt seem to work well across repos.

Reply:
Ok